src/Processors/Formats/IInputFormat.h

```diff
@@ -11,6 +12,65 @@ namespace DB
 
 using ColumnMappingPtr = std::shared_ptr<ColumnMapping>;
 
+class InputFormatErrorsLogger
```

Comment:
Let's move this to separate files `InputFormatErrorsLogger.h/InputFormatErrorsLogger.cpp`